src/test/kotlin/no/nav/familie/ba/soknad/api/personopplysning/PersonopplysningerServiceTest.kt

```diff
@@ -48,9 +50,45 @@ class PersonopplysningerServiceTest {
         assertTrue(person.barn.isEmpty())
     }
 
+    @Test
+    fun `borMedSøker skal returnere false når søkerAdressen er null` () {
+        val søkerAdresse = null;
+        val barneAdresse = Bostedsadresse(null, Matrikkeladresse(1, "E2", "tillegg", "1456", "1223"), null)
+        val borMedSøker = personopplysningerService.borMedSøker(søkerAdresse, barneAdresse)
+
+        assertFalse(borMedSøker)
+    }
+
+    @Test
+    fun `borMedSøker skal returnere true når adressene til barn og søker er like`(){
+        val barneAdresse = Bostedsadresse(null, Matrikkeladresse(1, "E2", "tillegg", "1456", "1223"), null)
+        val søkerAdresse = Bostedsadresse(null, Matrikkeladresse(1, "E2", "tillegg", "1456", "1223"), null)
+        val borMedSøker = personopplysningerService.borMedSøker(søkerAdresse, barneAdresse)
+
+        assertTrue(borMedSøker)
+    }
+
+    @Test
+    fun `borMedSøker skal returnere false når søker og barn har ulik adresse, men lik type`(){
+        val barneAdresse = Bostedsadresse(null, Matrikkeladresse(3, "E67", "tillegg", "1456", "1223"), null)
+        val søkerAdresse = Bostedsadresse(null, Matrikkeladresse(1, "E2", "tillegg", "1456", "1223"), null)
+        val borMedSøker = personopplysningerService.borMedSøker(søkerAdresse, barneAdresse)
+
+        assertFalse(borMedSøker)
+    }
+
+    @Test
+    fun `borMedSøker skal returnere false hvis søkerAdresse er ukjent`() {
+        val barneAdresse = Bostedsadresse(null, Matrikkeladresse(1, "E2", "tillegg", "1456", "1223"), null)
+        val søkerAdresse = Bostedsadresse(null, null, UkjentBosted("oslo"))
+        val borMedSøker = personopplysningerService.borMedSøker(søkerAdresse, barneAdresse)
+
+        assertFalse(borMedSøker)
+    }
+
     private fun settNavnOgRelasjonerFil(filNavn: String) {
-        every { client.hentNavnOgRelasjoner(any()) } returns
-                mapper.readValue(File(getFile("pdl/$filNavn.json")), PdlHentPersonResponse::class.java)
+        every { client.hentSøker(any()) } returns
```

Comment:
Fint å skille ut denne lesinga i egen funksjon 👍  For å tydeliggjøre i testene hva som skjer, kanskje vi kan bevare every i testene? Slik at det i hver test f.eks. blir noe som
`every { client.hentSøker(any()) } returns pdlMockFor("pdlPersonMedFlereRelasjoner")`